### ISSUE TITLE
Atmosphere package should say "meteor add semantic:ui flemay:less-autoprefixer" at very top.

### ISSUE THREAD

**Issue by giantelk** [Semantic-Org/Semantic-UI-Meteor#39]
At top of Atmosphere package page: https://atmospherejs.com/semantic/ui
It only shows `meteor add semantic:ui` Look at the field to the left of the Starred button.

Is should show `meteor add semantic:ui flemay:less-autoprefixer`.

**Comment by nooitaf** [comment]
That field gets auto-generated by atmosphere with the user and package name. We can't change that.
We could add the less-autoprefixer as dependency tho. But there might be a good reason why its not added yet.

**Comment by giantelk** [comment]
Ideally it Atmosphere package should behave like NPM packages and include
dependencies automatically just by adding the main package I.e. Meteor add
semantic:ui

**Comment by nooitaf** [comment]
As i said, there might be a good reason. Maybe @flemay can shed some light on this decision?

**Comment by flemay** [comment]
@giantelk @nooitaf Heya! Sorry for my late reply! :| 

I did first add `flemay:less-autoprefixer` as a dependency. I changed my mind later to make it as a dependency injection instead. I prefer the user to choose any library he wants to compile the less files instead of imposing one. Personal taste I guess. 

The first section in the README is about installation. Just hope sometimes people read the README first :)
